rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy

```diff
@@ -370,6 +373,115 @@ class ScheduledExecutionController  extends ControllerBase{
             )
         }
     }
+
+    @Get(uri = "/job/{id}/components")
```

Comment:
todo: let's also add `@Hidden` to exclude it from OpenAPI spec until it's no longer incubating